next-cmake/cmake/FindBLIS.cmake

```diff
@@ -0,0 +1,87 @@
+# Copyright Advanced Micro Devices, Inc., or its affiliates.
+# SPDX-License-Identifier: MIT
+
+include(FindPackageHandleStandardArgs)
+
+set(BLIS_PATH_4_2_0 "/opt/AMD/aocl/aocl-linux-gcc-4.2.0/gcc")
+set(BLIS_PATH_4_1_0 "/opt/AMD/aocl/aocl-linux-aocc-4.1.0/aocc")
+set(BLIS_PATH_4_0   "/opt/AMD/aocl/aocl-linux-aocc-4.0")
+
+# # NOTE: Keeping this code for reference of previous implementation
```

Comment:
I think this discussion's likely already been had, but are you set on only including these reference libs? Imo it would be better to search for the old libs until we figure out the aocl dependency/EULA issue.